docs/hello-world.md

```diff
@@ -44,7 +44,7 @@ module.exports = (app) => {
 };
 ```
 
-To get started, you can use the instructions for [Developing an App](./development.md) or remix this 'Hello World' project on Glitch:
+To get started, you can use the instructions for [Developing an App](https://probot.github.io/docs/development/) or remix this 'Hello World' project on Glitch:
```

Comment:
This is a bit simpler and would also work if probot moved to another domain (like probot.org)

```suggestion
To get started, you can use the instructions for [Developing an App](/docs/development/) or remix this 'Hello World' project on Glitch:
```